locale.c

```diff
@@ -8044,11 +8044,8 @@ S_ints_to_tm(pTHX_ struct tm * mytm,
 
         /* But since mini_mktime() doesn't handle these fields, save the
          * results we already have for them */
-#  ifdef HAS_TM_TM_GMTOFF
-        long int gmtoff = mytm->tm_gmtoff;
```

Comment:
Is this general enough that we don't have to use your `struct tm` method for this field too?  When writing this, I was surprised that this was a `long`, since I can't imagine it ever exceeding 14 * 60 * 60 (for double daylight time 12 timezones away from gmt).

Reply:
somehow I misread this until I downloaded it and looked on my box.  You already had done it the way I was suggesting.